config/initializers/new_framework_defaults.rb

```diff
@@ -0,0 +1,24 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+#
+# Read the Rails 5.0 release notes for more info on each option.
+
+# Enable per-form CSRF tokens. Previous versions had false.
+Rails.application.config.action_controller.per_form_csrf_tokens = true
```

Comment:
no idea ... suppose it works with both ...

On Sun, Jul 24, 2016 at 5:44 PM, Roman Sandler <notifications@github.com>
wrote:

> In config/initializers/new_framework_defaults.rb
> <https://github.com/zendesk/samson/pull/972#discussion_r71999403>:
>
> > @@ -0,0 +1,24 @@
> > +# Be sure to restart your server when you modify this file.
> > +#
> > +# This file contains migration options to ease your Rails 5.0 upgrade.
> > +#
> > +# Read the Rails 5.0 release notes for more info on each option.
> > +
> > +# Enable per-form CSRF tokens. Previous versions had false.
> > +Rails.application.config.action_controller.per_form_csrf_tokens = true
>
> works with form_tag or need to switch to form_for?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/972/files/679bed7d1a5aa027bee461b7fb9e0d65cd6a1dfa#r71999403>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ_WKRtuiSCLlFFEMIBSIzEMBVsS-ks5qZAbdgaJpZM4IdSiz>
> .
>